### DISCUSSION TITLE
how do i do a circular rebalance with a node?

### DISCUSSION THREAD

**Discussion by ralyodio** [Ride-The-Lightning/RTL#1329] [Troubleshooting Questions] [answered]
I entered 1k satoshis in amount to rebalance. It says next to it "Receive from peer" -- what do I put here? I tried copying the public id of a node, but then it just shows an error: "Receive from Peer is required."

How do I complete this? I'm unable to receive zaps until I move some funds apparently.

**Comment by saubyk** [comment, marked as answer]
Hi @ralyodio you need to have multiple channels on your node to execute a circular rebalance

A circular rebalance is a payment you make to yourselves, which sends payment out from one channel, and is received via another. This means that you need at least two channels on your node, one with liquidity to send the payment out and another with liquidity to receive the amount.

If you just have one channel or channels with only liquidity on your side (i.e. local balance), you won't be able to do circular rebalance. Also, you cannot put a random node's pubkey on circular rebalancing, you need to have a channel with the node with which you can do the rebalance. You must choose from the dropdown menu.

It sounds like you're trying to acquire inbound liquidity, so that you can receive payments on your node. There are multiple options for you to accomplish that:
1. Send some payments out from your node, so that you get the equivalent inbound capacity. You can setup a lightning wallet on your phone and send payments to your wallet from the node
2. Use loop-out to get inbound liquidity
3. Buy liquidity from an Lighting Service Provider (LSP)
4. Ask someone with a lightning node to open a channel to your node

Hope this helps
